auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -314,14 +314,28 @@ export default defineComponent({
       if (!documentKey || !documentName) return // safety check
 
       state.isDownloading = true
-      await BusinessService.downloadDocument(documentKey, documentName).catch(error => {
+      const documentClass = 'CORP'
+      try {
+        const docUrl: string = await BusinessService.getDownloadUrl(documentKey, documentClass)
+        const link = document.createElement('a')
```

Comment:
some Duplicated code, already exists in business.services.ts?

![image](https://github.com/user-attachments/assets/1450a391-75e8-40d5-9e0c-18f37644d67b)